Arsenstrus

ArsenStrus opened the pull request:
Hw4,5

ArsenStrus committed:
new file:   hw/hw04/arsenstrus/4.1_strus.py

ArsenStrus committed:
new file:   hw/hw05/arsenstrus/5.1_strus.py

ArsenStrus committed:
new file:   hw/hw06/arsenstrus/6.1_strus.py

ArsenStrus committed:
new file:   hw/hw03/arsenstrus/3.1_strus.py

ArsenStrus committed:
Merge branch 'main' into arsenstrus

lhalam reviewed (commented):
вида з гіта папку `.idea`
`git rm --cached .idea`

ArsenStrus committed:
Merge branch 'main' into arsenstrus

ArsenStrus committed:
deleted:    .idea/.gitignore
	deleted:    .idea/UA1217PythonFundamentals.iml
	deleted:    .idea/inspectionProfiles/profiles_settings.xml
	deleted:    .idea/misc.xml
	deleted:    .idea/modules.xml
	deleted:    .idea/vcs.xml

ArsenStrus committed:
new file:   hw/hw07/arsenstrus/7.1_strus.py
	new file:   hw/hw07/arsenstrus/7.2_strus.py

ArsenStrus commented:
Видалив ту папку ( не розумію як вона там взагалі опинилася )

ArsenStrus committed:
Merge branch 'main' into arsenstrus

ArsenStrus committed:
modified:   hw/hw07/arsenstrus/7.1_strus.py
	new file:   hw/hw08/arsenstrus/8.1/main.py
	new file:   hw/hw08/arsenstrus/8.1/models/__init__.py
	new file:   hw/hw08/arsenstrus/8.1/models/admin.py
	new file:   hw/hw08/arsenstrus/8.1/models/user.py
	new file:   hw/hw08/arsenstrus/8.1/utils/__init__.py
	new file:   hw/hw08/arsenstrus/8.1/utils/formatter.py
	new file:   hw/hw08/arsenstrus/8.1/utils/logger.py
	new file:   hw/hw08/arsenstrus/8.2/main.py
	new file:   hw/hw08/arsenstrus/8.2/password_check.py
	new file:   hw/hw08/arsenstrus/8.3/funtions.py
	new file:   hw/hw08/arsenstrus/8.3/main.py
	new file:   hw/hw09/arsenstrus/9.1/main.py
	new file:   hw/hw09/arsenstrus/9.2/main.py
	new file:   hw/hw09/arsenstrus/9.3/main.py

ArsenStrus committed:
new file:   hw/hw10/arsenstrus/10.1_strus.py
	new file:   hw/hw10/arsenstrus/10.2_strus.py
	new file:   hw/hw11/arsenstrus/11.1_strus.py
